### ISSUE TITLE
iOS Support

### ISSUE THREAD

**Issue by jahd2602** [FrogSquare/GodotFireBase#36]
Hello, thanks for the wonderful module!

I was wondering, is there any plan for iOS support? Or, at least, people interested on it?

Thanks!

**Comment by RameshRavone** [comment]
It's not that I didn't llan for IOS, its just I don't own a mac, I already coded about 40 - 50 percent of the module in object-c except I can't build them for now

**Comment by jahd2602** [comment]
@RameshRavone Is that 40-50% in a branch? I have an old macbook here to test.

**Comment by vini-guerrero** [comment]
Any updates on this?

**Comment by RameshRavone** [comment]
No, I broke admob and remoteconfig. And this is not a priority for me so it
would take a while.

**Comment by Triquetras** [comment]
Hello! Any updates on this?
